runtime/metricsview/astsql.go

```diff
@@ -74,7 +74,8 @@ func (b *sqlBuilder) writeSelectWithLabels(n *SelectNode) error {
 	return nil
 }
 
-func (b *sqlBuilder) writeSelect(n *SelectNode) error {
+func (b *sqlBuilder) writeSelect(n *SelectNode, level int) error {
+	b.out.WriteString(strings.Repeat("\t", level))
```

Comment:
Can we remove the pretty printing here? It's redundant extra length and allocations. If it's for debugging, there are many SQL formatters that can prettify a query (I use the `adpyke.vscode-sql-formatter` extension in VS Code)

Reply:
Meaning additional manual work in exchange for nano-optimisation

Reply:
I'm more worried about the need for understanding, maintaining and debugging formatting code for no reason. There are plenty of decent SQL formatter libraries for Go if we have cases where we need to print approximately formatted SQL.